lisa/testsuite.py

```diff
@@ -237,10 +237,26 @@ class TestCaseRequirement:
     os_type: Optional[search_space.SetSpace[Type[OperatingSystem]]] = None
 
 
-def simple_requirement(
+def _construct_node(
```

Comment:
It should be used in simple_requirement only, so it doesn't need to be a separated method.

Reply:
but it is also possible that user pass a none node?

Reply:
fix